hexa/workspaces/tests/test_schema/test_workspace.py

```diff
@@ -97,6 +97,13 @@ def setUpTestData(cls):
             role=WorkspaceMembershipRole.VIEWER,
             status=WorkspaceInvitationStatus.ACCEPTED,
         )
+        cls.INVITATION_BAR = WorkspaceInvitation.objects.create(
```

Comment:
It's a bit risky to reuse the same class variable here - maybe `cls.INVITATION_BAZ`?